stablehlo/tests/interpret_exponential.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: exponential_op_test_i64
+func.func @exponential_op_test_i64() -> tensor<4xf64> {
+  %operand = stablehlo.constant dense<[0.0, 1.0, -2.3, 3.1415926535897931]> : tensor<4xf64>
+  %result = stablehlo.exponential %operand : tensor<4xf64>
+  func.return %result : tensor<4xf64>
```

Comment:
Can we keep the test in the spec? That would affect the coverage but is more familiar to the user of the spec.

Reply:
Sure, aligned Spec with test. And also update them with f64: the highest precision of float type.